Unix/buildtool

```diff
@@ -811,8 +811,8 @@ if [ "$arg1" = "cflags" -o "$arg1" = "cxxflags" ]; then
 
             ## Optimization options.
             if test -z "$debug_opt"; then
-                test -n "$c_opt" && r="$r -g"
-                test -n "$cxx_opt" && r="$r -g -O"
+                test -n "$c_opt" && r="$r"
```

Comment:
Huh? You're saying, for the C compiler, make `r="$r"`. This won't do anything at all, will it?